### ISSUE TITLE
[QUESTION]

### ISSUE THREAD

**Issue by arcas0803** [dart-lang/angular_components#102]
Are you gone release content components like cards, toolbar, drawer, something like an app shell in polymer?

**Comment by nshahan** [comment]
No immediate plans to release these types of components in the near future. We have discussed but do not have general use versions ready to open source at the moment.

**Comment by zoechi** [comment]
There are community packages available in pub.dartlang.org at least for some of these components

**Comment by ligustah** [comment]
It's a little bit messy, but at least for a pet project I've used https://getmdl.io/ for the layout stuff (like menus, cards, tables).

**Comment by TedSander** [comment]
mdl is a great way to get pure styling/html things like nav-bar, and cards.

As of now I don't see us providing components for things that can be done purely with style as Components currently have performance costs. In the future we may do something better here, but no near term plans.

**Comment by madisona** [comment]
where the native components come in nicely are things like the layout. There is a lot of complication in the javascript just to show / hide the sidebar for example. It would be nice to not have to try to recreate things like that

**Comment by arcas0803** [comment]
I know every solution mentioned here, but i though that this implementation of material design would be all-in-one implementation, i mean  anything that you need to build in one single package.
Thanks for the response.

**Comment by ligustah** [comment]
Actually, I didn't have too many problems with that. I've used [js_facade_gen](https://github.com/dart-lang/js_facade_gen) and definitions from [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped) to interop with the mdl javascript. As I said, it really feels kind of messy, but with very little code it allowed me to use the spinner component from mdl before the native dart spinner was released, for example.
